backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -562,3 +413,126 @@ async def set_memory_provider(provider: str):
+    return p, {}
+
+
+async def get_memory_config_raw():
+    """Return the memory section of config.yml as YAML text for editing."""
+    try:
+        _, data = _find_config_yml_path()
+        memory_section = data.get("memory", {}) or {}
+        # Return as a full section including the root key for clarity
+        yaml_text = yaml.safe_dump({"memory": memory_section}, sort_keys=False)
+        return {"config_yaml": yaml_text}
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)